packages/theme/src/theme/_modals.scss

```diff
@@ -25,10 +55,17 @@
 	}
 
 	&-body {
+		flex-basis: auto;
 		flex-grow: 1;
+		flex-shrink: 1;
+		display: flex;
+		flex-direction: column;
+		overflow: hidden;
```

Comment:
I don't understand why we have those 3 properties above. Why do we style the content that can be whatever we want ?